packages/nocodb/src/filters/global-exception/global-exception.filter.ts

```diff
@@ -18,7 +18,8 @@ export class GlobalExceptionFilter implements ExceptionFilter {
     const ctx = host.switchToHttp();
     const response = ctx.getResponse<Response>();
 
-    // todo: error log
+    const logger = new Logger(GlobalExceptionFilter.name);
```

Comment:
We can keep it as a class property

```
private logger = new Logger(GlobalExceptionFilter.name);
```

Within method

```
this.logger.error(exception.message, exception.stack);
```

Reply:
updated (force-pushed to correct typo in commit message)